cmd/odf/mon/mon.go

```diff
@@ -0,0 +1,18 @@
+package mon
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// MonsCmd represents the mon command
+var MonsCmd = &cobra.Command{
+	Use:                "mons",
+	Short:              "Supports sub-commands like 'restore-quorum'",
+	DisableFlagParsing: true,
+	Hidden:             true,
```

Comment:
All of the mon commands are hidden? I was thinking only advanced commands like restore-quorum might need to be hidden.

Reply:
IIRC, Eran's comment was to have all the dev preview command hidden. I guess it is also mentioned in Jira by him

Reply:
Let's discuss this more. I see the point of hiding dev preview items, but also would prefer not to.